src/utils/line.ts

```diff
@@ -243,14 +345,41 @@ export const formatSecuredLineData = (
     [BigNumber.from(0), {}]
   );
 
-  const escrow = {
+  // Get escrow collateral events
+  const escrowCollateralEvents: CollateralEvent[] = _.flatten(
```

Comment:
wont need `flatten` and `merge` when u do escrow.events instead of escrow.deposits

Reply:
I need `flatten` and `merge` because you cannot query `escrow.events` directly from subgraph and know that they are escrow/collateral events.

happy to be corrected on this if I'm wrong!